card/card.css

```diff
@@ -0,0 +1,171 @@
+@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.2/font/bootstrap-icons.css");
+@import url(../styles.css);
+* {
+    font-family: sans-serif;
+}
+:root {
+    --main-box-shadow: 0 3px 10px rgb(0 0 0 / 0.2);
+    --hover-effect-color: #eeeded;
+    --light-gray: rgb(176, 176, 176);
+}
+
+.main-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    /* margin: 4rem; */
+    border: 1px solid red;
+}
+html {
+    font-size: 100%;
+}
+h2 {
+    font-size: 1.5rem;
+}
+h3 {
+    font-size: 1rem;
+}
+p {
+    font-size: 0.9rem;
+}
+
+.header {
+    opacity: 0.9;
+}
+img {
+    height: 100%;
+    width: 100%;
+}
```

Comment:
i don't think selecting using html elements is a good idea because if you import this stylesheet then all the img elements will be effected